src/DV7toDV8.sh

```diff
@@ -273,7 +460,73 @@ do
         rm "$DV8_RPU_BIN" 
         rm "$DV8_BL_RPU_HEVC"
     fi
+    
+    # Check if remuxing was successful
+    if [[ -f "$tempDir/$mkvBase.DV8.mkv" ]]; then
```

Comment:
With all of the filenames in this if-branch now being used 3+ times between here and their earlier usage, it feels like the Rule of 3 applies and it could be time to DRY them up into named variables. What do you think?